src/Startup.fs

```diff
@@ -26,7 +26,7 @@ type Startup private () =
         app
             .UseGiraffeErrorHandler(errorHandler)
             .UseWebSockets()
-            .UseMiddleware<GraphQLWebSocketMiddleware<Root>>(Schema.executor, fun () -> { RequestId = Guid.NewGuid().ToString() })
```

Comment:
This is annoying, but do we know if we need websockets for gql subscriptions? I personally rather have the playground around for easier testing and find a solution to this when we know we need it.